


service OrderService {

    void receipt_domestic_transfer(
    1: i64 package_id, 2: i64 operator),


}
